Sources/Scheduler.swift

```diff
@@ -13,6 +13,48 @@ import Foundation
 	import CDispatch
 #endif
 
+extension Date {
+	internal func addingTimeInterval(_ interval: DispatchTimeInterval) -> Date {
+		return addingTimeInterval(interval.asTimeInterval())
+	}
+}
+
+extension DispatchTimeInterval {
+	internal func asTimeInterval() -> TimeInterval {
```

Comment:
@NachoSoto would you keep the `asTimeInterval` name? I've not seen precedent for this pattern yet outside of Apple's use of `cgColor` (i.e. using the type name itself) except it'd be strange to call a `DispatchTimeInterval` method called `timeInterval`. 😛